### PR TITLE
Edits for clarity and personal foibles

### PR DESCRIPTION
**Pull request by low-sky**

### CODE REVIEW

**Comment by low-sky** [comment]
whoops! some more commits coming

File: main.tex [hunk 1]
```diff
@@ -366,22 +354,15 @@ \subsection{Plotting}
...
+types (e.g., ammonia and formaldehyde hyperfine models, the \hh rotational ladder, and recombination line models).
 
 In radio and millimeter astronomy, there are several molecular line groups that
 consist of several Gaussian profiles separated by a fixed frequency offset.
 These hyperfine line groups are often unique probes of physical parameters
-because in many cases the ``main'' line can become optically thick, while the
-``satellite'' lines remain thin, meaning that the line optical depth can be
-measured with a single spectrum.  \pyspeckit provides the \texttt{hyperfine}
-model class to handle this class of molecular line transitions, and it includes
+because these different features have different, known relative optical depths. In this case, the measured relative amplitudes of these different features allows the optical depth to be measured from a single spectrum.  \pyspeckit provides the \texttt{hyperfine} model class to handle this class of molecular line transitions, and it includes
```
Comment (keflavich) [inline main.tex:365]: allows->allow

File: main.tex [hunk 2]
```diff
@@ -576,10 +542,10 @@ \section{Parameter Error Estimation for Ammonia}
...
 The parameter values and errors are shown in the upper right.  The associated
 error estimate triangle diagram is shown in Figure \ref{fig:nh3parerrestdemo}.
-The correct parameters are $T_{R}=T_{ex}=35$, $N=15$, $\sigma=2$, and $v=0$,
-all of which are reasonably recovered.  However, note that $T_{ex} > T_{rot}$
-is generally nonphysical, yet the allowed parameter space for $T_{ex}$ includes
-such values.
+The correct parameters are $T_{\mathrm{R}}=T_{\mathrm{ex}}=35$, $N=15$, $\sigma_v=2$, and $v=0$,
+all of which are reasonably recovered.  However, note that $T_{\mathrm{ex}} > T_{\mathrm{R}}$
+is generally nonphysical, yet the allowed parameter space for $T_{\mathrm{ex}}$ includes
+such values. \textcolor{red}{EWR: Not showing the empty spectral region would make a nicer visualization, IMHO.}
```
Comment (keflavich) [inline main.tex:548]: You mean the space between the 1-1 and 2-2 lines?  It's just inconvenient to code that up, and I do need to show both... I guess I could use the wrapper and make a two-panel figure...

File: main.tex (hunk 1 above)
Comment (keflavich) [reply main.tex:365]: (I've fixed this in the master branch)

**Pull request merged**